### PR TITLE
 feat(temporarily-restores-xva): Add function to test a XVA file restoration

### PR DESCRIPTION
**Pull request by vgallissot**
Added:
* temporarily-restores-xva command, to be able to test a file restoration and delete the resulting VM right away
* export_metadata function, needed for the above restoration, so we know infos about the VM and in which SR to reimport it
* WORKING_DIR var, to enhance files creation and deletion between functions so they don't conflict with each other
* import_vm function now exports an envvar when success, containing the ID of the newly created VM (usefull for new command temporarily-restores-xva)
* export_vm function now exports an envvar when success, containing the path to the XVA (usefull for new command temporarily-restores-xva)
* command `archive` now uses function `temporarily-restores-xva` to ensure XVA file is restorable

Changed:
* breaking: LOCAL_XVA_DIR envvar has been renamed to LOCAL_TMP_DIR. Also default value `/tmp/xva_dir` has been changed to `/tmp/xo-archiver`
* breaking: AWS_BUCKET_XVA_ARCHIVES envvar has been renamed to AWS_BUCKET_XO_ARCHIVES
* export command now export metadata on top of XVA
* some comments so they better fit reality
* fix bug where the request to add credentials were not displayed when calling the script with import or delete
* fix bug where credentials were not asked for in some cases, because using the xo_cli_wrapper instead of the raw binary

### CODE REVIEW

**Review by christian-vdz — APPROVED** [review]
LGTM 🚀

File: xo-archiver [hunk 1]
```diff
@@ -258,58 +272,127 @@ function xo_register {
...
-    if xo_cli_wrap vm.import sr=$1 @=$2
+    # using xo-cli and not wrapper here, because we redirect stdout (incompatible with ask-for-pass)
+    # still, we try to renew auth to avoid expired credential error
+    renew_xo_cli_auth
+    if xo-cli vm.import sr=$1 @=$2 > $2-imported.json
     then
-        log "Import completed"
+        # This new VM-ID is necessary to run temporarily_restores_xva function
+        # but it also improves the output log
+        export IMPORTED_VM_ID=$(cat $2-imported.json | jq --exit-status -r '.result')
```
Comment (jmenan) [inline xo-archiver:386]:
```suggestion
        export IMPORTED_VM_ID=$(jq --exit-status -r '.result' '$2-imported.json' )
```

**Review by tanguyfalconnet — COMMENTED** [review]
Why don't add execute permission on script ?

**Review by QuentinBtd — APPROVED** [review]

**Comment by vgallissot** [comment]
> Why don't add execute permission on script ?

Added :)

**Pull request merged**